modules/util/portal-tools-soy-builder/src/main/java/com/liferay/portal/tools/soy/builder/maven/BaseSoyJsMojo.java

```diff
@@ -50,7 +49,7 @@ public void execute() throws MojoExecutionException {
 
 				String[] includedFiles = scanner.getIncludedFiles();
 
-				if (ArrayUtil.isNotEmpty(includedFiles)) {
+				if (includedFiles.length > 0) {
```

Comment:
Haha, this is what I did previously with BuildCSSMojo and ArrayUtil was added as dependency during a review.  i agree with this simplification